Source/ACE.DatLoader/Entity/MotionData.cs

```diff
@@ -0,0 +1,45 @@
+﻿using ACE.Entity;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ACE.DatLoader.Entity
+{
+    public class MotionData
+    {
+        public byte Bitfield { get; set; }
```

Comment:
Seems like Bitfield is not used?  Perhaps comment `/// <summary>Unknown usage</summary>`
Similarly, Bitfield2 is used, perhaps make an enumeration for it and give it a name?